### PR TITLE
Add new notice

### PR DESCRIPTION
**Pull request by SAnatoliiS**
image
image

### CODE REVIEW

File: kit/popups/src/UIInteractiveNotice/Actions.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    return <View style={styles.container}>{actionList.map(Action)}</View>;
+}
+
+const styles = StyleSheet.create({
+    container: {
+        flexDirection: 'row',
+        flexWrap: 'wrap',
+        marginTop: UILayoutConstant.contentInsetVerticalX1,
+        marginHorizontal: -UILayoutConstant.contentOffset / 2,
+        marginBottom: -UILayoutConstant.contentInsetVerticalX2,
```
Comment (sertony) [inline kit/popups/src/UIInteractiveNotice/Actions.tsx:43]: Why do we have to use negative margins here?

File: kit/popups/src/UIInteractiveNotice/CloseButton.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    if (!showCloseButton) {
+        return null;
+    }
+    return (
+        <View style={style}>
+            <UIPressableArea
+                onPress={onClose}
+                style={{
+                    marginVertical: -UILayoutConstant.contentInsetVerticalX4,
+                    marginHorizontal: -UILayoutConstant.contentOffset,
```
Comment (sertony) [inline kit/popups/src/UIInteractiveNotice/CloseButton.tsx:24]: Same question. Besides, does it work well with Android where all the Views have overflow: 'hidden'?

**Review by sertony — APPROVED** [review]

File: kit/popups/src/UIInteractiveNotice/Actions.tsx (hunk 1 above)
Comment (SAnatoliiS) [reply kit/popups/src/UIInteractiveNotice/Actions.tsx:43]: To make a large tap zone without remake all padding layout principles.

File: kit/popups/src/UIInteractiveNotice/CloseButton.tsx (hunk 2 above)
Comment (SAnatoliiS) [reply kit/popups/src/UIInteractiveNotice/CloseButton.tsx:24]: It works well on all platforms, I checked.

**Review by vetkamak — APPROVED** [review]

**Pull request merged**